plugin/storage/grpc/shared/grpc_server_test.go

```diff
@@ -142,7 +158,7 @@ func TestGRPCServerFindTraces(t *testing.T) {
 
 		var traces []*model.Trace
 		var traceID model.TraceID
-		var trace *model.Trace
+		trace := &model.Trace{}
```

Comment:
null ptr as initial value seems safer in the test

Reply:
I use code from `grpcClient.GetTrace` https://github.com/jaegertracing/jaeger/blob/9e9bf6ad39eab7526b29eebe9fd14c3e6542e5c8/plugin/storage/grpc/shared/grpc_client.go#L91. This code must be equal. We have to fix it in two places if `grpcClient.GetTrace` contains bug.